src/Nethermind/Nethermind.Merge.Plugin/Handlers/GetPayloadV3Handler.cs

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Threading.Tasks;
+using Nethermind.Core;
+using Nethermind.Core.Extensions;
+using Nethermind.JsonRpc;
+using Nethermind.Logging;
+using Nethermind.Merge.Plugin.BlockProduction;
+using Nethermind.Merge.Plugin.Data;
+
+namespace Nethermind.Merge.Plugin.Handlers
+{
+    /// <summary>
+    /// <a href="https://github.com/ethereum/execution-apis/blob/main/src/engine/experimental/blob-extension.md#engine_getpayloadv3">engine_getpayloadv3</a>
+    /// </summary>
+    public class GetPayloadV3Handler : IAsyncHandler<byte[], GetPayloadV2Result?>
```

Comment:
very similar code in all GetPayloadVXHandler's, maybe we should do some generic base class?